README.md

```diff
@@ -34,6 +34,7 @@ To solve the preceding issues, this tool works as follows.
 
 * This tool does not guarantee the atomicity of deletion. If you access the rows that are being deleted, you will get the inconsistent view of the database.
 * This tool does not delete rows which were inserted while the tool was running.
+* This tool does not support scenarios involving foreign key constraints with 'ON DELETE CASCADE' in cases where there is a circular dependency among the tables. As a result, it may not be able to truncate the tables.
```

Comment:
If I understand correctly, we have at least two scenarios that don't fully work.

1. If there is a circular dependency among the tables, truncation will be failed.
2. If user uses `--exclude-tables` only for the referencing table that has `ON DELETE CASCADE`, that table will be (surprisingly) truncated by cascade-deletion of the referenced table.

If it's correct, we can say something like...

* This tool does not support truncating tables that use foreign key constraints in some scenarios:
  * If there is a circular dependency among the tables, truncation will be failed.
  * If `--exclude-tables` is used only for the referencing table that has `ON DELETE CASCADE`, that table will be truncated by cascade-deletion of the referenced table.

Reply:
> If --exclude-tables is used only for the referencing table that has `ON DELETE CASCADE`, that table will be truncated by cascade-deletion of the referenced table.

If we need to present detailed scenarios, We may be able to add the following scenario as well.

- If `--tables` is used for a table referenced with `ON DELETE CASCADE`, the table will be truncated with the referrer tables.

Reply:
Sounds good! Let's use the consistent wording `referencing table` and `referenced table` as they are used in the public documentation: https://cloud.google.com/spanner/docs/foreign-keys/overview#how-to-define-foreign-keys

Reply:
> the consistent wording `referencing table` and `referenced table` 

This sounds easy to understand. I adopt this.